spring-integration-smb/README.md

```diff
@@ -92,8 +92,8 @@ NOTE: Setting of the SMB protocol min/max versions must be done in your implemen
 
 ````java
 @Bean
-public SmbSessionFactory smbSessionFactory(new myCIFSContext()) {
-    SmbSessionFactory smbSession = new SmbSessionFactory();
+public SmbSessionFactory smbSessionFactory() {
+    SmbSessionFactory smbSession = new SmbSessionFactory(new myCIFSContext());
```

Comment:
```suggestion
    SmbSessionFactory smbSession = new SmbSessionFactory(new MyCIFSContext());
```

Reply:
I mean it not so usual for Java to name classes from lower case.
It might be OK for your project, but in general let's don't encourage people for bad practice.

Reply:
Yeah agreed, however it's just an example snippet so I'm assuming people won't follow it religiously :-o. I'm good either way.